File: gamespage/src/components/Game.js
const Game = ( {id, name, price, image} ) => {

    return (
        <article>
            <h3>{ name } (id: {id})</h3>
            <p>Price: {price}</p>
            <img src={`https://localhost:5001/images/${image}`}/>
        </article>


    )



}

export default Game;

File: GamesApi/Models/Game.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace GamesApi.Models{

    public class Game{

        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }
        public string Name { get; set; }
        public int Price { get; set; }
        public string Image { get; set; }

    }

}